prow/config/jobs/enhancements-1.25.yaml

```
@@ -0,0 +1,353 @@
+branches:
+- release-1.25
+env:
+- name: BUILD_WITH_CONTAINER
+  value: "0"
+- name: BUILD_WITH_CONTAINER
+  value: "0"
+image: gcr.io/istio-testing/build-tools:release-1.25-ba6e7d3a0ff9666b1cc2a650507b0d99acfefff4
+jobs:
+- command:
+  - ../test-infra/scripts/validate_schema.sh
+  - --document-path=./features.yaml
+  - --schema-path=./features_schema.json
+  env:
+  - name: BUILD_WITH_CONTAINER
+    value: "0"
+  - name: BUILD_WITH_CONTAINER
+    value: "0"
```

```suggestion
  env:
  - name: BUILD_WITH_CONTAINER
    value: "0"
```
Remove duplicate entry as described in "legacy" manual steps for Step 3 in https://github.com/istio/istio/issues/54707